Solutions/README.md

```diff
@@ -1 +1,122 @@
-Guidelines (coming soon)
+# Azure Sentinel Solutions 
+Azure Sentinel Solutions provide an in-product experience for central discoverability, single-step deployment, and enablement of end-to-end product and/or domain and/or vertical scenarios in Azure Sentinel. This experience is powered by Azure Marketplace for Solutions’ discoverability, deployment and enablement and Microsoft Partner Center for Solutions’ authoring and publishing. 
+
+# Why Azure Sentinel Solutions?
+* Customers can easily discover packaged content and integrations that deliver product value or for a domain or for a vertical centrally in Azure Sentinel. 
+* Customers can easily deploy and optionally enable content to get started immediately.
+* Providers or partners can deliver combined product or domain or vertical value via Solutions in Azure Sentinel and be able to productize investments too.
+
+# Solution Package
+Clone the repository [Sentinel-AzMP-Solutions](https://msazure.visualstudio.com/One/_git/Sentinel-AzMP-Solutions) to C:\One.
+## Create Input File
+Create an input file and place it in the path C:\One\Sentinel-AzMP-Solutions\solutions\automation\input.
+
+**Input File Format:**
+```
```

Comment:
Would probably be good to specify that this interface defines the structure of the json input file. So that they don't go and start creating .TS interfaces.